src/hotspot/share/classfile/systemDictionary.cpp

```diff
@@ -1398,8 +1477,77 @@ void SystemDictionary::clear_invoke_method_table() {
     }
   }
 }
+
+bool SystemDictionary::should_not_dump_class(InstanceKlass* k) {
```

Comment:
Add log here to help us understand why many classes cannot be dumped by CDS

Reply:
Done.